.env.sample

```diff
@@ -1,5 +1,6 @@
 # ################# REQUIRED ENV VARS START #################
 PORT=8080
+MONGODB_PORT=10000 # mongodb port for e2e testing
```

Comment:
Don't set a new value for port, take entire mongodb url from env as it already exisits

Reply:
I'm using MongoMemoryServer it's require PORT we can change variable name to make it more clear like **MONGO_MEMORY_SERVER_PORT** what do you think? 

Reply:
Sure, that would be better! 🙂👍🏻